src/Common/RadixSort.h

```diff
@@ -252,7 +252,7 @@ struct RadixSort
         /// There are loops of NUM_PASSES. It is very important that they are unfolded at compile-time.
 
         /// For each of the NUM_PASSES bit ranges of the key, consider how many times each value of this bit range met.
-        CountType histograms[HISTOGRAM_SIZE * NUM_PASSES] = {0};
```

Comment:
Here only the first element was zero-initialized before. It's sad our sanitizers didn't catch that.

Reply:
1. No... Otherwise it cannot work at all.
AFAIK C and C++ allows to omit array elements in initializer and all the rest will be zero initialized. Similar to `arr = {}`.

2. Memory Sanitizer can catch uninitialized memory for sure.

Reply:
Yes, you're right. I've checked it, the documentation says "the remaining elements are set to their default values (which for fundamental types, means they are filled with zeroes)".